Calculate/Calculator.cs

```diff
@@ -0,0 +1,72 @@
+using System.Globalization;
+namespace Calculate;
+
+public class Calculator
+{
+    public IReadOnlyDictionary<char, Func<int, int, double>> MathematicalOperations {
+        get;
+    } = new Dictionary<char, Func<int, int, double>> {
+        { '+', Add },
+        { '-', Subtract },
+        { '*', Multiply },
+        { '/', Divide }
+    };
+
+    public static double Add(int a, int b)
+    {
+        return a + b;
+    }
+
+    public static double Subtract(int a, int b)
+    {
+        return a - b;
+    }
+
+    public static double Multiply(int a, int b)
+    {
+        return a * b;
+    }
+
+    public static double Divide(int a, int b)
+    {
+        return (double)a / b;
+    }
+    
+    public bool TryCalculate(string input)
```

Comment:
You'll need an out parameter here. e.g.
```suggestion
    public bool TryCalculate(string input, out result)
```